examples/rollbar-spring-webmvc/src/main/java/com/example/springwebmvc/ExampleApplication.java

```diff
@@ -0,0 +1,43 @@
+package com.example.springwebmvc;
+
+import static com.rollbar.notifier.config.ConfigBuilder.withAccessToken;
+
+import com.rollbar.notifier.Rollbar;
+import com.rollbar.notifier.config.Config;
+import com.rollbar.spring.webmvc.RollbarExceptionResolver;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+
+@SpringBootApplication
+@Configuration()
+@ComponentScan()
+public class ExampleApplication {
+  public static void main(String[] args) {
+    SpringApplication.run(ExampleApplication.class, args);
+  }
+
+  /**
+   * Handles all exceptions with Rollbar's Exception Resolver.
+   */
+  @Bean
+  public HandlerExceptionResolver rollbarExceptionResolver() {
+
+    // If you use a configBuilder the framework is set to spring-web-mvc
+    return new RollbarExceptionResolver(withAccessToken("POST-SERVER-ACCESS-TOKEN")
```

Comment:
I think we could extract to another method the creation of the `Rollbar` bean. That would make it accessible to Spring DI, and will allow to inject it in the controllers.

Also, we would have it available to create the `RollbarExceptionResolver` by declaring it as a dependency in this method.

```
@Bean
public Rollbar rollbar() {
    return withAccessToken("POST-SERVER-ACCESS-TOKEN").build();
}

@Bean
public HandlerExceptionResolver rollbarExceptionResolver(Rollbar rollbar) {
    return new RollbarExceptionResolver(rollbar);
}
```

Reply:
done